flit/inifile.py

```diff
@@ -93,7 +94,7 @@ def verify_classifiers(classifiers):
     _verify_classifiers_cached(classifiers)
 
 
-def read_pkg_ini(path):
+def read_pkg_ini(path, develop=False):
```

Comment:
I don't think the options you pass in should reach this level - the `inifile` module is to read and check the config file, and that process shouldn't be affected by what you want to use it for.